docs/sources/guides/whats-new-in-v7-1.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "What's New in Grafana v7.1"
+description = "Feature and improvement highlights for Grafana v7.1"
+keywords = ["grafana", "new", "documentation", "7.1", "release notes"]
+type = "docs"
+[menu.docs]
+name = "Version 7.1"
+identifier = "v7.1"
+parent = "whatsnew"
+weight = -16
++++
+
+# What's new in Grafana v7.1
+
+This topic includes the release notes for the Grafana v7.1, which is currently in beta. For all details, read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+The main highlights are:
+
+- **
+
+## Unified querying experience in Explore
+TODO
+
+## Query history search
+In Grafana v 7.1 we are introducing search functionality in Query history. You can search across queries and your comments. It is especially useful in combination with a time filter and data source filter. Read more about Query history [here]({{ < relref ”../features/explore/index.md#query-history” >}}).
+
+## Provisioning of apps
+
+Grafana v7.1 adds support for provisioning of app plugins. This allows app plugins to be configured and enabled/disabled using configuration files. Read more about provisioning of app plugins [here]({{ < relref ”../administration/provisioning.md#plugins” >}}).  
+
+## Azure Monitor Datasource
+
+Support for multiple dimensions has been added to all services in the Azure Monitor datasource. This means you can now group by more than one dimension with time series queries. With the Kusto based services, Log Analytics and Application Insights Analytics, you can also select multiple metrics as well as multiple dimensions.
+
+Additionally, the “Raw Edit” mode for Application Insights Analytics has been replaced with a new service in the drop down for the datasource and is called “Insights Analytics”. The new query editor behaves in the same way as Log Analytics.  
+
+## Influx Datasource
+
+Support for Flux and Influx v2 has been added.
+
+## Google Stackdriver datasource has been renamed to Google Cloud Monitoring
+
```

Comment:
```suggestion
```

I have changed my mind; maybe we can skip this since it will be included in the Changelog. Sorry for the indecision.